servicebroker/lib/servicebroker.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var async = require('asyncawait/async');
+var await = require('asyncawait/await');
+var model = require(path.join(__dirname, './model'))();
+
+var ServiceBroker = function ServiceBroker() {};
+ServiceBroker.prototype.getCatalog = function() {
+  var catalog = JSON.parse(
+    fs.readFileSync(path.join(__dirname, '../config/catalog.json'), 'utf8'));
+  return catalog;
+};
+
+ServiceBroker.prototype.provisionService = async(function(serviceId, orgId, spaceId) {
```

Comment:
Just want to make sure #1,  the findOrCreate returns a Promise, so we need to handle it with promise way findOrCreate.then()  as well?  The await/async wrap works correctly, but we did need to add await/async wrapper for each invocation.   All the other comments will be addressed.